kuma/core/middleware.py

```diff
@@ -166,7 +166,9 @@ def process_response(self, request, response):
         urlconf = getattr(request, 'urlconf', settings.ROOT_URLCONF)
 
         # Kuma: assume locale-prefix patterns, including default language
-        if response.status_code == 404 and not language_from_path:
+        if (response.status_code == 404 and
+                not language_from_path and
+                not request.path_info.startswith("/api/")):
```

Comment:
👍 Good catch.

Reply:
This doesn't seem right to me. We have other 'no locale' paths that don't need a special case, and we've stopped using ``LANGUAGE_URL_IGNORED_PATHS`` in the last 12 months. I'm seeing if there is a better place for this logic, but I can fix in a later PR if I'm slow.